ambari-server/src/main/python/ambari_server/setupSecurity.py

```diff
@@ -809,8 +816,8 @@ def setup_ldap(options):
   print 'Review Settings'
   print '=' * 20
   for property in ldap_property_list_reqd:
-    if ldap_property_value_map.has_key(property):
-      print("%s: %s" % (property, ldap_property_value_map[property]))
+    if ldap_property_value_map.has_key(property.prop_name):
+      print("%s %s" % (property.ldap_prop_val_prompt, ldap_property_value_map[property.prop_name]))
```

Comment:
Maybe logger.info ?

Reply:
Everything else is being displayed by the print; I just modified the content we check/display and not the way how we display it.